examples/eager_op_multithread/CMakeLists.txt

```diff
@@ -5,10 +5,13 @@ find_library(TENSORFLOW_LIB tensorflow HINT $ENV{HOME}/libtensorflow2/lib)
 find_package(Threads REQUIRED)
 
 set(CMAKE_CXX_STANDARD 17)
+#set(CMAKE_CXX_STANDARD 20)
+
 
 #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer -fsanitize=thread")
 #set(CMAKE_LINKER_FLAGS "${CMAKE_LINKER_FLAGS} -fno-omit-frame-pointer -fsanitize=thread")
 
 add_executable(example main.cpp)
 target_include_directories(example PRIVATE ../../include $ENV{HOME}/libtensorflow2/include)
-target_link_libraries(example "${TENSORFLOW_LIB}" Threads::Threads)
+target_link_libraries (example "${TENSORFLOW_LIB}")
```

Comment:
Please keep `Threads::Threads`. This does not compile on LInux because `pthread` is required.

Reply:
Fixed 👍

Reply:
Can you revert all changes to this file? It makes this commit cleaner and focuses on the span header. You can create separate testing files to show it works with and without C++20.

Reply:
target_link_libraries (example "${TENSORFLOW_LIB}") is required because it reports link error on my macos.

Reply:
What is your cmake version? `Threads::Threads` was introduced in cmake 3.1, and cmake 3.10 is requested here. This specific test case is related to threading, so `Threads::Threads` has to be linked.

I strongly recommend creating a separate test case for the span class and do not touch unrelated test case.